src/scripts/sync_subclassof.py

```diff
@@ -309,7 +309,8 @@ def sync_subclassof(
     missing_ancestor_rels = rels_indirect_mondo_mondo.union(rels_direct_mondo_mondo).difference(ancestors_mondo_mondo)
     if missing_ancestor_rels:
         rels_for_printing =  [rel for rel in list(missing_ancestor_rels)[:5]]
-        logging.error(
+        # TODO: temp:
+        raise ValueError(
```

Comment:
@joeflack4 it seems you are right - when I reran it now, with -B, the error was not thrown. Must have been a syncing issue!